index.html

```diff
@@ -6,14 +6,64 @@
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css\style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+<body class="body">
+    <header class="header">
+     <nav class="navigation">
+        <img class="logo" src="img\karma-logo.svg" alt="Karma Logo">
+        <label class="toggle">&#9776;</label>
+        <input type="checkbox" id="toggle">
+        
+        <section class="Nav">
+                <a href="#" class="Nav1">Meet Karma</a>&nbsp; &nbsp;
+                <a href="#" class="Nav2">How it Works</a> &nbsp; &nbsp;
+                <a href="#" class="Nav3">Store</a>&nbsp; &nbsp;
+                <a href="#" class="Nav4">Blog</a>&nbsp; &nbsp;
+                <a href="#" class="Nav5">Help</a>&nbsp; &nbsp;
+                <a href="#" class="Nav6">Login</a>&nbsp; &nbsp;
+            </section>
+           
+        </nav>
+    </header>
+    <article>
+        <img class="backgroundimage" src="img\first-background.jpg" alt="">
+        <section class="maintitle">
+            <h1 class="maintitle">Introducing Karma</h1>
+            <p class="maintitle">Bring Wifi with you, everywhere you go.</p>
+            <a href="#"><button class="maintitle" class="button">Learn More</button></a>
+        </section>
+    </article>
+    
+    
+    <main class="main">
+        <section class="headline">
+            <h1>Everyone needs a little Karma.</h1>
+        </section>
+        <section class="device">
+            <img src="img\icon-devices.svg" class="icons" alt="devices icon">
+            <h3>Internet for all devices</h3>
+        </section>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+        <section lass="coffee">
+            <img src="img\icon-coffee.svg" class="icons" alt="Coffee icon">
+            <h3>Boost your productivity</h3>
+        </section>  
+
+        <section class="fuel">
+            <img src="img\icon-refill.svg"  class="icons" alt="fuel tang icon">
+            <h3>Pay as You Go</h3>
+        </section>&nbsp;  
+    </main>
+
+    <footer class="footer">
+        <h5>Join us on</h5>
+        <img  class="footer-icons" src="img\twitter-icon.svg" alt="Twitter Logo">&nbsp;&nbsp;
+        <img class="footer-icons" src="img\facebook-icon.svg" alt="Facebook Logo">&nbsp;&nbsp;
+        <img class="footer-icons" src="img\instagram-icon.svg" alt="Instagram Logo">&nbsp;&nbsp;
```

Comment:
It's worth adding an empty link to these, just to give the impression that you can click on them when hovering over 🙂